backend/entry.ts

```diff
@@ -4,10 +4,10 @@ import {Angular2Adapter} from './adapters/angular2';
 let channel = {
   sendMessage: (message) => {
     console.log('Inspected script sending: ', message);
-    return window.postMessage({
+    return window.postMessage(JSON.parse(JSON.stringify({
```

Comment:
It's (a bit of hack) to strip out the methods of the object being passed via postMessage, since they would cause a DataCloneError on certain directive components. If there's a more legit way to do it, I'd love to know.